script/build.js

```diff
@@ -101,7 +116,11 @@ if (options.buildtype === 'production') {
   ignoreList.push('profile/*');
   ignoreList.push('auth/*');
   ignoreList.push('education/apply-for-education-benefits-new');
+} else if (options.buildtype === 'staging') {
+  ignoreList.push('education/apply-for-education-benefits/application.md');
+  ignoreList.push('education/apply-for-education-benefits-new');
```

Comment:
Shouldn't this be everything _except_ the education benefits, if that's what this PR is meant to allow end-to-end testing of?